api/src/main/java/ca/bc/gov/educ/api/batchgraduation/controller/JobLauncherController.java

```diff
@@ -846,6 +848,48 @@ public ResponseEntity<BatchJobResponse> launchEdwSnapshotJob(@RequestBody Snapsh
         }
     }
 
+    @PostMapping(EducGradBatchGraduationApiConstants.EXECUTE_YEARLY_ARCHIVE_STUDENTS_RUN_BATCH_JOB)
+    @PreAuthorize(PermissionsConstants.RUN_GRAD_ALGORITHM)
```

Comment:
Definitely it is not running Graduation Algorithm run.  Is there any other permission more like it?

Reply:
RUN_GRAD_ALGORITHM is used on every endpoint in the controller. I'll add "archive" one but we have to review most other  endpoints as well

Reply:
Updated